backend/src/src-schedule/src/main/java/com/bosch/osmi/sw360/schedule/service/ScheduleHandler.java

```diff
@@ -42,16 +46,17 @@ private boolean wrapSupplierException(SupplierThrowingTException body, String se
             try {
                 return body.get();
             } catch (TException e) {
-                log.error("was not able to schedule sync for client with name:" + serviceName + " message:" + e.getMessage(), e);
+                log.error("Was not able to schedule sync for client with name:" + serviceName + " message:" + e.getMessage(), e);
                 return RequestStatus.FAILURE;
             }
         };
         return Scheduler.scheduleNextSync(wrappedBody, serviceName);
     }
 
     @Override
-    public RequestSummary scheduleService(String serviceName, User user) throws TException {
-        if (!PermissionUtils.isAdmin(user)){
+    public RequestSummary scheduleService(String serviceName) throws TException {
```

Comment:
Because the SchedulingServlet has to call ` handler.scheduleService(cleanServiceName);` automatically in the autostart-method on startup.